### DISCUSSION TITLE
Browse takes me to a subfolder root folder not selectable

### DISCUSSION THREAD

**Discussion by Beemer2** [WinMerge/winmerge#2044] [General]
I am a first time user.  I want to compare an MP3 folder that has subfolders to a FLAC folder that has the same name subfolders.  When I browse the MP3 folder it immediately takes me to a subfolder instead of the top level folder 'MP3'.  The menu indicates that I can include subfolders but I don't want to be taken to a specific subfolder which is what is happening just now.  There does not appear to be a way for me to select top level folders.

**Comment by sdottaka** [comment]
As shown in the image below, if you press the Open button without selecting a subfolder within the MP3 folder, the MP3 folder will be selected.

If that doesn't work, you can drag and drop the folder into the text box labeled `1st File or Folder` or `2nd File or Folder`.

![image](https://github.com/WinMerge/winmerge/assets/98126/170622fb-82a0-4209-99ef-f14a9a5b6205)

**Comment by Beemer2** [comment]
Many thanks!
